docs/docs/draft/documentation/README.md

```diff
@@ -1312,12 +1312,76 @@ curl -XDELETE http://localhost:5681/meshes/mesh-1/traffic-logs/tl-1
+
+#### Kubernetes
+On Kubernetes deployments, the process is automated by leveraging [ServiceAccountToken](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#service-account-automation) that is mounted in every pod.
+
+#### Universal
+
+On Universal deployments, you have to generate a token before starting a dataplane. Dataplane Token is a [JWT tokens](https://jwt.io) that contains the _name_ and _mesh_ of the dataplane.
+It is signed by RSA private key generated on the first run of the control plane. Tokens are not stored in the control plane,
+the only thing that is stored is a private key that is used to verify if a token is valid. 
+
+You can generate token either by REST API
+```bash
+curl -XPOST http://localhost:5679/tokens --data '{"name:" "dp-echo-1", "mesh": "default"}'
+```
+
+or by using `kumactl`
+```bash
+kumactl generate dataplane-token --name=dp-echo-1 --mesh=default > /tmp/kuma-dp-echo1-token
+``` 
+
+The token should be stored in a file and then used when starting `kuma-dp`
+```bash
+$ kuma-dp run \
+  --name=dp-echo-1 \
+  --mesh=default \
+  --cp-address=http://127.0.0.1:5682 \
+  --dataplane-token=/tmp/kuma-dp-echo-1-token
+```
+
+##### Accessing Dataplane Token Server from a different machine
+
+By default, the Dataplane Token Server is exposed only on localhost. If you want to generate tokens from a different machine than control plane you have to secure the connection:
+1) Generate certificate for the HTTPS Dataplane Token Server and set via `KUMA_DATAPLANE_TOKEN_SERVER_PUBLIC_TLS_CERT_FILE` and `KUMA_DATAPLANE_TOKEN_SERVER_PUBLIC_TLS_KEY_FILE` config environment variable.
+2) Pick a public interface on which HTTPS server will be exposed and set it via `KUMA_DATAPLANE_TOKEN_SERVER_PUBLIC_INTERFACE`.
+   Optionally pick the port via `KUMA_DATAPLANE_TOKEN_SERVER_PUBLIC_PORT`. By default, it will be the same as the port for the HTTP server exposed on localhost.
+3) Generate one or more certificates for the clients of this server. Pass the path to certificates via `KUMA_DATAPLANE_TOKEN_SERVER_PUBLIC_CLIENT_CERT_FILES`.
+4) Configure `kumactl` with client certificate. `kumactl config control-planes add --name local-kuma --address http://kumacp.internal:5681 --dataplane-token-client-cert cert.pem --dataplane-token-client-key key.pem`
+
+### mTLS
+
+Once the connection between the dataplane and SDS server is established, the dataplane can now fetch its certificate
+and certificates of dependant services' dataplanes. When establishing a connection between services each side validates
+each other dataplane certificate confirming the identity. Every connection is denied by default, so you have to explicitly allow it using [TrafficPermission](/docs/DRAFT/policies/#traffic-permissions).
```

Comment:
> Every connection is denied by default, so you have to explicitly allow it using [TrafficPermission](/docs/DRAFT/policies/#traffic-permissions).

It's not a part of mTLS flow.

Let's make it clear by moving into a separate section.